### ISSUE TITLE
Support for Ledger Stax

### ISSUE THREAD

**Issue by siy** [LedgerHQ/ledger-device-rust-sdk#78]
It would be great to have this device supported by SDK.

**Comment by tdejoigny-ledger** [comment]
FYI this topic is in progress

**Comment by siy** [comment]
how #114 relates to #109 ?

**Comment by agrojean-ledger** [comment]
@siy #114 is the follow-up to #109 but I preferred opening a new PR. So it is directly related.

However everything in those PRs is still pretty much a work in progress and not yet ready to be release (but we want to go forward with it asap).

**Comment by siy** [comment]
> However everything in those PRs is still pretty much a work in progress and not yet ready to be released (but we want to go forward with it asap).

I'm following closely and working on Stax version of our app in parallel.
